src/tchap/util/TchapRoomUtils.ts

```diff
@@ -38,15 +37,19 @@ export default class TchapRoomUtils {
      * @returns string that matches of one TchapRoomAccessRule //todo or null? or empty?
      */
     static getTchapRoomAccessRule(room: Room): TchapRoomAccessRule {
-        return room.currentState.getStateEvents(TchapRoomAccessRulesEventId, "")?.getContent().rule;
+        // the state event TchapRoomAccessRulesEventId is working only when the rule access is invite -> maybe a change in the backend configuration ? needs to check
```

Comment:
hmmm. If some rooms have no value for TchapRoomAccessRulesEventId ("im.vector.room.access_rules") it means that they were not created by a tchap client. (or the tchap client was in an experimental phase, most likely web v4 in its early stages :) )

In this case, the room could have any type of configuration (if it was created by an element client for example), and I don't know if we can decide on one of the 3 tchap-types. 🤔